elastic/apm-server
# Service metrics txn events txn metrics

## Motivation/summary



Add `event.success_count` to transaction events, transaction metrics and service metrics. It was previously available for service metrics at `transaction.success_count` by #9791 


## Checklist




## How to test these changes



1. Check that `event.success_count` is correct for transaction event, transaction metrics and service metrics
2. Check that they work with avg aggregation

## Related issues


Closes #5243
